.symfony.cloud.yaml

```diff
@@ -6,44 +6,41 @@ runtime:
     extensions:
         - apcu
         - mbstring
-        
+        - iconv
+        - ctype
 
 build:
     flavor: none
 
 web:
     locations:
         "/":
-            root: "web"
+            root: "public"
             expires: 1h
-            passthru: "/app.php"
+            passthru: "/index.php"
 
 disk: 2048
 
 mounts:
-    "app/cache": { source: local, source_path: "app/cache" }
-    "app/logs": { source: local, source_path: "app/logs" }
+    "var/cache": { source: local, source_path: "var/cache" }
+    "var/logs": { source: local, source_path: "var/logs" }
 
 hooks:
     build: |
         set -x -e
 
-        # Remove the development front-controller if present
-        (>&2 rm web/app_dev.php)
-
-        SYMFONY_ENV=prod composer install --prefer-dist --optimize-autoloader --classmap-authoritative --no-progress --no-ansi --no-interaction --no-dev
-        (>&2 SYMFONY_ENV=prod app/console cache:clear --no-warmup)
-        (>&2 SYMFONY_ENV=prod app/console cache:warmup)
+        APP_ENV=prod composer install --prefer-dist --optimize-autoloader --classmap-authoritative --no-progress --no-ansi --no-interaction --no-dev
+        (>&2 SYMFONY_ENV=prod bin/console cache:clear --no-warmup)
+        (>&2 SYMFONY_ENV=prod bin/console cache:warmup)
 
         # Keep the cache in a persistent directory
         # If your cache can be read-only, you can skip this step
-        (>&2 mkdir -p tmp/cache && mv app/cache/prod tmp/cache/)
+        (>&2 mkdir -p tmp/cache && mv var/cache/prod tmp/cache/)
 
     deploy: |
         set -x -e
 
         # "install" cache
         # If your cache can be read-only, you can skip these steps
-        rm -rf app/cache/prod
-        #cp -Rp tmp/bootstrap.php.cache app/bootstrap.php.cache
-        cp -Rp tmp/cache/prod app/cache/
+        rm -rf var/cache/prod
+        cp -Rp tmp/cache/prod var/cache/
```

Comment:
You need to use this snippet instead: https://github.com/symfonycorp/cloud-templates/blob/master/01-silex.yaml#L34-L47

Reply:
I'm not familiar with what is going on in this file. I limited my changes to obvious ones (like changing `app/cache` to `var/cache`, or removing the front-controller since there isn't a "dev" front-controller anymore). I'm not sure I'm the right person to make changes to the existing behavior. Especially considering I can't test those changes.